### DISCUSSION TITLE
Arrangement of input and output differs

### DISCUSSION THREAD

**Discussion by Dinock** [GDQuest/learn-gdscript#139] [General]
In the lessons' text the output is always on the left side while the code example/input is always on the right side.
But in the practice screen it is the other way around: The code is always on the left side and the output is always shown on the right side.

I don't think this is a big deal to worry about but it does make the examples and practices look a tiny bit inconsistent. So instead of filing an issue I thought it would be more helpful to discuss this.

Screenshot lessons' text:
![lesson_text](https://user-images.githubusercontent.com/54438803/149020030-42ea4316-ceff-42ec-998f-da9001e77f3b.png)

Screenshot practice screen:
![practice_screen](https://user-images.githubusercontent.com/54438803/149020156-cedc680e-994a-49d6-a60c-33a644d35a36.png)

**Comment by NathanLovato** [comment]
The code editor is centered in the practice screen because it's the central part of that interface.

In the lesson, people read left to right, so we put the interactive code example on the left to try first and the code listing on the right to read the code afterwards.
